runtime/protocol/http-client-engines/http-client-engine-ktor/common/src/aws/smithy/kotlin/runtime/http/engine/ktor/DefaultHttpClientEngine.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package aws.smithy.kotlin.runtime.http.engine.ktor
+
+import aws.smithy.kotlin.runtime.http.engine.*
+
+/**
+ * Specifies the common default http client of which platform specific [HttpClientEngine]'s actualize
+ *
+ * @param config Provides configuration for the DefaultHttpClientEngine
+ */
+expect class DefaultHttpClientEngine(config: HttpClientEngineConfig = HttpClientEngineConfig.Default) : HttpClientEngineBase
```

Comment:
Renamed from "Ktor" as it's not certain that all actuals will be a ktor implementation.